lib/vagrant-libvirt/util/network_util.rb

```diff
@@ -1,6 +1,14 @@
 require 'nokogiri'
 require 'vagrant/util/network_ip'
 
+class IPAddr
```

Comment:
I think should require `ipaddr` for this file because we're explicitly opening the IPAddr class to modify it's behaviour here.

Reply:
Oh, yes, sure. I missed this dependencie.  I'll fix it ASAP.